apps/a1/Dockerfile

```
@@ -0,0 +1,33 @@
+# Pre-requisite: uv sync --all-packages
+
+# Build
+# We use the same image as the runtime base image so that Python
+# version and location matches. Might save on data transfer costs.
+FROM us-west1-docker.pkg.dev/serverless-runtimes/google-22/runtimes/python313 AS builder
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)